docs/faq.md

```diff
@@ -756,3 +772,59 @@ On examining the container structure with [Dive](https://github.com/wagoodman/di
 The user had used Jib's ability to install extra files into the image ([Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#adding-arbitrary-files-to-the-image), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#adding-arbitrary-files-to-the-image)) to install a library file by placing it in `src/main/jib/lib/libfoo.so`. This would normally cause the `libfoo.so` to be installed in the image as `/lib/libfoo.so`. But `/lib` and `/lib64` in the user's base image were symbolic links. Jib does not follow such symbolic links when creating the image. And at container initialization time, Docker treats these symlinks as a file, and thus the symbolic link was replaced with `/lib` as a new directory. As a result, none of the system shared libraries were resolved and dynamically-linked programs failed.
 
 Solution: The user installed the file in a different location.
+
+## Jib CLI 
+ 
+### How does the `jar` command support Standard JARs?
+The Jib CLI supports both [thin JARs](https://docs.oracle.com/javase/tutorial/deployment/jar/downman.html) (where dependencies are specified in the JAR's manifest) and fat JARs.
+
+However, the current limitation of using a fat JAR is that the embedded dependencies will not be placed into the designated dependencies layers.
+They will instead be placed into the classes or resources layer. We hope to have better support for fat JARs in the future.
```

Comment:
How about something like this?

```suggestion
However, the current limitation of using a fat JAR is that the embedded dependencies will not be placed into the designated dependencies layers. They will instead be placed into the classes or resources layer. Therefore, for efficiency, we recommend against containerizing fat JARs (Spring Boot fat JARs are an [exception](and link to the FAQ below?)) if you can prepare thin JARs. We hope to have better support for fat JARs in the future.
```

And just saying, if you do want to render the second sentence as a new paragraph, you should put an empty line between before it. If starting a new paragraph was not your intention, I suggest not putting the second sentence in the next line to avoid confusion.

Reply:
I think you missed the part "Therefore, for efficiency, we recommend against containerizing fat JARs (Spring Boot fat JARs are an [exception](and link to the FAQ below?)) if you can prepare thin JARs."

Reply:
great catch, just updated this.